missas/core/templates/home.html

```diff
@@ -0,0 +1,215 @@
+                    <p class="card-text small text-muted mb-0">Horários verificados</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- Estados e Cidades -->
+    <div class="row mb-4">
+        <div class="col-12">
+            <h3 class="mb-3" style="color: var(--primary-blue); font-weight: 600;">
+                <i class="fa-solid fa-map me-2"></i>
+                Estados e Cidades com Paróquias
+            </h3>
+            <div class="row">
+                {% for state in states_with_cities %}
+                    <div class="col-12 col-md-6 col-lg-4 mb-3">
+                        <div class="card h-100 border-0 shadow-sm">
+                            <div class="card-body">
+                                <h5 class="card-title"
+                                    style="color: var(--primary-blue);
+                                           font-weight: 600">
+                                    <a href="/{{ state.slug }}/"
+                                       class="text-decoration-none"
+                                       style="color: var(--primary-blue)">{{ state.name }}</a>
+                                </h5>
+                                <div class="cities-list">
+                                    {% for city in state.cities_with_parishes %}
+                                        {% if forloop.counter <= 5 %}
+                                            <a href="/{{ state.slug }}/{{ city.slug }}/"
+                                               class="badge bg-light text-dark me-1 mb-1 text-decoration-none"
+                                               style="border: 1px solid var(--gray-200)">{{ city.name }}</a>
+                                        {% endif %}
+                                    {% endfor %}
+                                    {% if state.cities_with_parishes|length > 5 %}
+                                        <span class="text-muted small">... e mais {{ state.cities_with_parishes|length|add:"-5" }}</span>
+                                    {% endif %}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                {% endfor %}
+            </div>
+        </div>
+    </div>
+    <!-- Projeto Open Source -->
+    <div class="row mb-4">
+        <div class="col-12">
+            <div class="alert alert-success border-0"
+                 style="background: linear-gradient(135deg, #E8F5E8, #C8E6C9);
+                        border-left: 4px solid #4CAF50">
+                <div class="row align-items-center">
+                    <div class="col-12 col-md-8">
+                        <h4 class="alert-heading mb-2" style="color: #2E7D32; font-weight: 600;">
+                            <i class="fa-brands fa-github me-2"></i>
+                            Projeto Open Source
```

Comment:
GitHub should be the last section. Make it less flashy; remove the green background.